src/clr-angular/accordion/_accordion.clarity.scss

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('accordion.clarity') {
+  .clr-accordion {
+    display: block;
```

Comment:
Most of our components use `display: flexbox`. So why did we opt for `display: block` in the stepper/accordion component? I actually don't know why we use flexbox most of the time.

Reply:
@mathisscott you probably know that answer to my question? 👆

Reply:
`.clr-accordion` doesn't need to be `display: flex`. `display: flex` allows an element's children to gobble up its full width _or_ height in more or less proportional slices. Here, accordion is the container or wrapper for children that use flex to apportion out the horizontal width to the container's grand-children.